src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -37,10 +49,12 @@ export class ColorPicker extends React.Component<ColorPickerProps> {
               onChange,
               key: index,
               className: classes.item,
+              name: this.groupName,
```

Comment:
I think it would be better to add a `name` prop
If it is undefined then we can use the aria-label as the name.
meaning:

```tsx
name: this.props.name || this.props['aria-label'] || this._groupName
```

Also, please use `_groupName` instead of `groupName`

Reply:
Done.